app.js

```diff
@@ -0,0 +1,15 @@
+const express = require("express");
+const { getAllTopics } = require("./controllers/topics.controllers");
+
+const app = express();
+
+app.get("/api/topics", getAllTopics);
+
+// Generic errors
+app.use((err, req, res, next) => {
+  if (err.status) {
+    res.status(err.status).send({ msg: err.msg });
+  }
+});
+
```

Comment:
Why have you included this error handling middleware, I can't see any point in your code so far where you're throwing a custom error. You also have no error test. I'd say the only error you should put it in at this point is a 500 internal server error to act as the final error middleware

Reply:
See this from the bottom of task 2 
![image](https://github.com/CraigAT1181/backend-project/assets/61753159/c76b7e9a-6032-4e27-a2b6-c35c4834c2a3)